procurement_auto_create_group/tests/test_auto_create.py

```diff
@@ -0,0 +1,247 @@
+# Copyright 2017-2020 ForgeFlow, S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestProcurementAutoCreateGroup(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```

Comment:
@AlexPForgeFlow FYI, you can also use `BaeCommon` class to avoid this line : https://github.com/odoo/odoo/blob/18.0/odoo/addons/base/tests/common.py#L20